### PR TITLE
Change docs title to install flower

### PR DESCRIPTION
**Pull request by danielnugraha**
Change docs title to “Install Flower” and add redirects

### CODE REVIEW

**Review by tanertopal — COMMENTED** [review]
@danieljanes this looks good to me. Am I missing something?

File: doc/source/install-flower.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Installing Flower
+Install Flower
 =================
```
Comment (danieljanes) [inline doc/source/install-flower.rst:2]: Change the `===` to match the line length of the previous line

**Comment by danieljanes** [comment]
@tanertopal, I added one comment, apart from that it looks good to me

**Review by danieljanes — APPROVED** [review]
Lgtm!

**Pull request merged**